scripts/build-and-test-postgres-wal-database-id.sh

```diff
@@ -0,0 +1,11 @@
+#! /bin/bash
+
+set -e
+
+export DATABASE=postgres
+export MODE=wal
+export TEST_MODULE=eventuate-local-java-cdc-connector-postgres-wal
+export TEST_CLASS=io.eventuate.local.postgres.wal.PostgresWalBinlogEntryReaderMessageTableTest
```

Comment:
Why do these profiles `postgres,PostgresWal` need to be activated when running `PostgresWalBinlogEntryReaderMessageTableTest` which is specific to postgres,PostgresWal?

Reply:
Chris, you probably ask why I added "postgres" to:

`@ActiveProfiles("${SPRING_PROFILES_ACTIVE:PostgresWal}")`

we need to use profiles like: "postgres, mssql" to activate properties that used to connect to those databases.

But PostgresWal is used only for postgres, and adding postgres as activated profile can't break something, but helps run that script from Idea. (I don't need to add it as env variable)

However postgres can be used with polling, so it does not make sense to have `application-postgres-wal.properties`, because we anyway need to have also `application-postgres.properties` with the same content.

Reply:
No. My point is that @ActiveProfiles on `PostgresWalBinlogEntryReaderMessageTableTest` can activate the needed profiles.
There is no need to specify SPRING_PROFILES_ACTIVE when executing PostgresWalBinlogEntryReaderMessageTableTest

Reply:
Oh.. I see now, yes, I will remove it